src/components/Personalization/createAutoRenderingHandler.js

```diff
@@ -0,0 +1,39 @@
+import addRenderToExecutedDecisions from "./utils/addRenderToExecutedDecisions";
+
+export default ({
+  viewCache,
+  executeDecisions,
+  executeCachedViewDecisions,
+  showContainers
+}) => {
+  return ({ viewName, pageWideScopeDecisions, formBasedComposedDecisions }) => {
```

Comment:
I recommend we rename those to something more generic that would apply to ODE, AJO...

`formBasedComposedDecisions` & `pageWideScopeDecisions`

Maybe something like: `renderableDecisions` & `nonRenderableDecisions`.

Also, aren't we renaming `decisions` to `propositions`?

Reply:
Thank you, Joe. I will rename those. 
Yes we return `decisions` the way we did earlier and newly `propositions` with decisions with rendered flag.

Reply:
@ninaceban are we going to rename those props?

Reply:
I agree. I now know that "formBased" => non auto renderable and "domAction" => auto renderable. I think it makes things more clear to use renderable and nonRenderable.